setup.cfg

```diff
@@ -1,2 +1,2 @@
 [bdist_wheel]
-universal = 1
+universal = 0
```

Comment:
I think it would be better to just remove the whole `[bdist_wheel]` section.

Reply:
Makes sense, thanks!